integrations/airtable/syncs/bases.md

```diff
@@ -0,0 +1,47 @@
+# Bases
+
+## General Information
+
+- **Description:** List all bases
+- **Version:** 0.0.1
+- **Group:** Others
+- **Scopes:**: schema.bases:read
+- **Endpoint Type:** Sync
+- **Code:** [🔗](https://github.com/NangoHQ/integration-templates/tree/main/integrations/airtable/syncs/bases.ts)
```

Comment:
might be clearer to display `[github.com](...)`

Reply:
Fixed in 3df2277a